controllers/chat/group/removeGroupMember.js

```diff
@@ -16,6 +16,13 @@ const removeGroupMember = async (req, res) => {
       const {newChannelName, betterChannelMember, betterChannelMemberObject, updatedChannel} =
         await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
 
+      await channel.sendMessage({
+        text: 'Removed a member',
+        type: 'system',
+        user_id: userId,
+        userAffectedId: targetUserId
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code sends a system message when a member is removed from the group. However, it lacks error handling. If `channel.sendMessage` fails for any reason (network issues, API limits, etc.), the function will throw an error and stop execution. Consider wrapping this in a try-catch block to handle potential errors gracefully.

```diff
+      try {
      await channel.sendMessage({
        text: 'Removed a member',
        type: 'system',
        user_id: userId,
        userAffectedId: targetUserId
      });
+      } catch (error) {
+        console.error(`Failed to send system message: ${error}`);
+      }
```
This way, even if sending the message fails, the rest of the function can continue executing.


<!-- This is an auto-generated comment by OSS CodeRabbit -->